package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "translation",
+  "version": "0.0.0",
+  "type": "module",
+  "engines": {
+    "node": ">=18.19.1"
```

Comment:
we only support 20+

Reply:
ok will have to try and up it and see how that goes

Reply:
✅